src/compounds/hero-card/src/index.tsx

```diff
@@ -0,0 +1,46 @@
+/** @jsx jsx */
+
+import * as React from 'react'
+import { jsx, Styled } from 'theme-ui'
+import { ButtonLink } from '@uswitch/trustyle.button-link'
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+  title: string
+  ctaText: string
+  children: React.ReactNode
+}
+
+const styles = (element?: string) =>
+  `compounds.heroCard.variants${element ? `.${element}` : ''}`
+
+const HeroCard: React.FC<Props> = ({ title, ctaText, children }) => {
+  return (
+    <div
+      sx={{
+        display: 'grid',
```

Comment:
In general we don't use grid (it's only used for product table right now as it was to only way to achieve that) due to it's limited support on IE11.  In general I would use flexbox where possible over grid, but if there's a specific reason for using it that's cool?

Reply:
Chatted on a call and this is a good use case for grid. Will be tested in IE11 to confirm everything works